Source/WebKit/Shared/Cocoa/XPCEndpointClient.h

```diff
@@ -30,7 +30,18 @@
 #include "WKDeclarationSpecifiers.h"
 #include <wtf/Lock.h>
 #include <wtf/OSObjectPtr.h>
-#include <wtf/spi/darwin/XPCSPI.h>
+
+#ifndef __XPC_H__
+#if OS_OBJECT_USE_OBJC
+OS_OBJECT_DECL(xpc_object);
```

Comment:
This also doesn't work, you can't redeclare API from a different module, that will result in an error if both modules are seen.

Reply:
Ah, yes, that is a good point. We cannot assume the XPC header will not have been included at this point.

Thanks for reviewing!